diff --git a/node_modules/solid-start-aws/entry.mjs b/node_modules/solid-start-aws/entry.mjs
index 4098897..ea318e1 100644
--- a/node_modules/solid-start-aws/entry.mjs
+++ b/node_modules/solid-start-aws/entry.mjs
@@ -19,10 +19,10 @@ export async function handler(event) {
     headers[name] = value;
   }
   if (response.headers.has('set-cookie')) {
-		const header = /** @type {string} */ (response.headers.get('set-cookie'));
-		// @ts-expect-error
-		headers['set-cookie'] =  splitCookiesString(header);
-	}
+    const header = /** @type {string} */ (response.headers.get('set-cookie'));
+    // @ts-expect-error
+    headers['set-cookie'] = splitCookiesString(header);
+  }
 
   return {
     statusCode: response.status,
@@ -33,7 +33,7 @@ export async function handler(event) {
 
 function createRequest(event) {
   const url = new URL(
-    event.rawPath,
+    event.path,
     `https://${event.requestContext.domainName}`
   );
 
@@ -43,7 +43,7 @@ function createRequest(event) {
   }
 
   const init = {
-    method: event.requestContext.http.method,
+    method: event.requestContext.httpMethod,
     headers,
   };
 
